app/controllers/beasts_controller.rb

```diff
@@ -1,2 +1,44 @@
 class BeastsController < ApplicationController
-end
+      rescue_from ActiveRecord::RecordInvalid, with: :render_unprocessable_entity_response
+      rescue_from ActiveRecord::RecordNotFound, with: :render_not_found_response
+
+      def render_unprocessable_entity_response(exception)
+            render json: exception.record.errors, status: :unprocessable_entity
+      end
+        
+      def render_not_found_response(exception)
+            render json: { error: exception.message }, status: :not_found
+      end
```

Comment:
For right now, let's focus on the controller methods and their request specs.  This falls more in line with validations and setting up a reaction from rails if you don't get a valid response which we will cover tomorrow.